.github/workflows/RELEASE.yaml

```diff
@@ -43,6 +47,114 @@ jobs:
+              <server>
+                <id>central</id>
+                <username>\${env.MAVEN_USR}</username>
+                <password>\${env.MAVEN_PSW}</password>
+              </server>
+              <server>
+                <id>gpg.passphrase</id>
+                <passphrase>\${env.MAVEN_GPG_PASSPHRASE}</passphrase>
+              </server>
+            </servers>
+          </settings>
+          EOF
+          
+      - name: Configure git user
+        run: |
+          # https://github.com/actions/checkout/issues/13#issuecomment-724415212
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Set Connectors release version
+        run: mvn -B versions:set-property -Dproperty=revision -DnewVersion=${RELEASE_VERSION} -DgenerateBackupPoms=false
+        env:
+          RELEASE_VERSION: ${{ github.event.inputs.version }}
+
+      - name: Deploy artifacts to Artifactory and Maven Central
+        run: mvn -B compile generate-sources source:jar javadoc:jar deploy -PcheckFormat -Psonatype-oss-release
+        env:
+          NEXUS_USR: ${{ steps.secrets.outputs.ARTIFACTORY_USR }}
+          NEXUS_PSW: ${{ steps.secrets.outputs.ARTIFACTORY_PSW }}
+          MAVEN_USR: ${{ steps.secrets.outputs.MAVEN_CENTRAL_DEPLOYMENT_USR }}
+          MAVEN_PSW: ${{ steps.secrets.outputs.MAVEN_CENTRAL_DEPLOYMENT_PSW }}
+          MAVEN_GPG_PASSPHRASE: ${{ steps.secrets.outputs.MAVEN_CENTRAL_GPG_SIGNING_KEY_PASSPHRASE }}
+
+      - name: Commit and tag
+        run: |
+          git commit -am "ci: release version ${RELEASE_VERSION}"
+          git tag -a ${RELEASE_VERSION} -m "ci: release version ${RELEASE_VERSION}"
+        env:
+          RELEASE_VERSION: ${{ github.event.inputs.version }}
+
+      - name: Set next development patch version
+        run: |
+          mvn -B versions:set-property -Dproperty=revision -DnewVersion=${NEXT_VERSION} -DgenerateBackupPoms=false
+          git commit -am "ci: set next development version"
+          git push --follow-tags
+        env:
+          NEXT_VERSION: ${{ github.event.inputs.nextVersion }}
+
+      - name: Get number of tags
+        id: get-tags
+        run: |-
+          TAG_COUNT=`git tag | wc -l`
+          echo "::set-output name=count::${TAG_COUNT}";
+
+      - name: "Build Changelog"
+        if: steps.get-tags.outputs.count > 1
+        id: changelog
+        uses: Requarks/changelog-action@v1
+        with:
+          token: ${{ github.token }}
+          tag: ${{ github.event.inputs.version }}
+          writeToFile: false
+          excludeTypes: build,docs,other,style,ci
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
+        with:
+          body: ${{ steps.changelog.outputs.changes }}
+          tag_name: ${{ github.event.inputs.version }}
+          files: |
+            **/THIRD_PARTY_NOTICES
+            **/element-templates/*.json
```

Comment:
There already is something like this at the bottom. We need to consolidate. Question to be answered: How do we add the collected element templates to the release? This is done by the `Bundle element templates` step. This needs some adjustment, I think. So we actually simply fetch the templates from the function modules. Currently, we base this on the `bash/Dockerfile` which doesn't make sense anymore.